sqlmesh/dbt/manifest.py

```diff
@@ -398,6 +399,9 @@ def _extra_dependencies(self, target: str, package: str) -> Dependencies:
         for call_name, node in extract_call_names(target, cache=self._calls):
             if call_name[0] == "config":
                 continue
+            elif isinstance(node, jinja2.nodes.Getattr):
```

Comment:
We already have a similar logic couple lines above for `adapter`:
```
 if call_name[0] != "adapter":
```
can we consolidate?

Reply:
no this is done for type checking, because getattr doesn't have the same argument thing as Call